common/scala/src/main/scala/org/apache/openwhisk/core/connector/MessageConsumer.scala

```diff
@@ -213,7 +213,13 @@ class MessageFeed(description: String,
       outstandingMessages = outstandingMessages.tail
 
       if (logHandoff) logging.debug(this, s"processing $topic[$partition][$offset] ($occupancy/$handlerCapacity)")
-      handler(bytes)
+      handler(bytes).andThen {
+        {
+          case Failure(e) =>
+            logging.error(this, s"Failed to process message for topic $topic : $e  (stack trace included)")
+            e.printStackTrace()
```

Comment:
printing the stack this way excludes it from the logger formatting - is that intended?

Reply:
not intentional - can you refer me to an example of better formatting for stack trace?
Thanks!

Reply:
Isn't it already part of the log line above anyway? I guess the broader question is: Why is the stack trace even needed here?

Reply:
Now it is, and I forgot to remove the `e.printStackTrace()` when I changed the log line to include it... 😔  fixing that...